app/providers/list/page.tsx

```
@@ -0,0 +1,127 @@
+          aboutSnippet: "Sustainable landscaping and garden design with eco-friendly practices and native plant expertise."
+        }
+      ],
+      total: 3,
+      page: page,
+      pageSize: pageSize
+    };
+  }
+}
+
+async function ProvidersList({ page, pageSize }: { page: number; pageSize: number }) {
+  const { providers, total } = await getProvidersList(page, pageSize);
```

ESLint

[error] 'total' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)